tools/travis_script.py

```diff
@@ -19,13 +19,8 @@ def build_iotjs(buildtype, args=[], env=[]):
 
 if __name__ == '__main__':
     test = os.getenv('OPTS')
-    if test == 'host-linux':
+    if test == 'jerryscript':
         build_jerry()
-        for buildtype in BUILDTYPES:
-            ex.check_run_cmd('./tools/build.py', [
-                '--cmake-param=-DENABLE_MODULE_ASSERT=ON',
-                '--run-test',
-                '--no-check-valgrind'])
```

Comment:
Why removing these builds?

Reply:
since `ENABLE_MODULE_ASSERT` is unused at all, it's duplicated with one another matrix.